plugins/code-scout/agents/scout.md

```
@@ -1,139 +1,32 @@
-
-### Find FastAPI Starter
-
-```
-User: "I need a production-ready FastAPI starter"
-
-Scout:
-1. gh search repos "fastapi boilerplate production" --sort stars
-2. gh search repos "awesome fastapi" --sort stars
-3. Analyze top 3, compare features
-4. Recommend best fit
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)